app/models/manageiq/providers/ansible_playbook_workflow.rb

```diff
@@ -14,6 +14,9 @@ def launch_runner
     kwargs = options.slice(:credentials, :hosts, :verbosity, :become_enabled)
 
     Ansible::Runner.run_async(env_vars, extra_vars, playbook_path, kwargs)
+  rescue => err
+    _log.warn("Failed to run #{playbook_path}: #{err}")
+    nil
```

Comment:
I assume this part is really trying to capture the missing executable exception as described in the issue?  I wonder if it's better to check for a) the presence/execution of ansible-runner ahead of time and b) allow this to blow up and gracefully handle things at the caller.  I'm concerned this may hide real exceptions.

Reply:
Hm if I catch the exception in `#execute` I can `signal_abort` in one place with the exception message would that help?